<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QMainWindow" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TicTacToe Optionen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBoxSymbol2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>8</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Spieler1: Name:                                                                                         Symbol:


Spieler2: Name:                                                                                         Symbol:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>18</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>58</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSymbol1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveClose">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save &amp;&amp; Close</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>checkBoxSymbol2</zorder>
   <zorder>lineEditName1</zorder>
   <zorder>lineEditName2</zorder>
   <zorder>checkBoxSymbol1</zorder>
   <zorder>saveClose</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
